.travis.yml

```diff
@@ -1,12 +1,13 @@
-language: node_js
-node_js:
-- '0.10'
+language: dart
+dart:
+- stable
+with_content_shell: true
```

Comment:
Changing the language to dart automatically uses the Dart Test Runner.  But, in order for the dart test runner to work, all the other lifecycle events need to have their dart defaults run.  Those tasks can be seen in https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/dart.rb ...